.github/workflows/ci.yml

```diff
@@ -137,3 +137,25 @@ jobs:
           PY_COLORS: '1'
           ANSIBLE_FORCE_COLOR: '1'
         shell: bash
+  lint:
+    runs-on: ubuntu-latest
+    needs: git-check
+    strategy:
+      fail-fast: false
```

Comment:
I propose having lint and git-check as one job, and the name would be lint.
All the other tests would depend on lint